marklogic-data-hub/src/main/java/com/marklogic/hub/impl/FlowManagerImpl.java

```diff
@@ -264,11 +265,9 @@ protected void deleteRelatedStepArtifacts(Flow flow, Step removedStep) {
             return;
         }
 
-        Path custModulePath = this.hubConfig.getModulesDir().resolve("root").resolve("custom-modules")
-            .resolve(removedStep.getStepDefinitionType().toString().toLowerCase())
-            .resolve(removedStep.getStepDefinitionName());
+        StepDefinition.StepDefinitionType stepTypeOfRemovedStep = EnumUtils.getEnumIgnoreCase(StepDefinition.StepDefinitionType.class, removedStep.getStepDefinitionType().toString());
```

Comment:
We can match stepdefintion type directly without invoking a third party library util via the getStepDefinitionType static method on the stepdefinition interface